### ISSUE TITLE
Passing setZone to luxon.Interval.fromISO yields "Invalid unit setZone" when parsing duration

### ISSUE THREAD

**Issue by dmalan** [moment/luxon#852]
**Describe the bug**

When using Luxon 1.25.0, passing the [`setZone`](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#static-method-fromISO) option to [`luxon.Interval.fromISO`](https://moment.github.io/luxon/docs/class/src/interval.js~Interval.html#static-method-fromISO), when parsing a duration, yields:

```js
duration.js:324 Uncaught Error: Invalid unit setZone
    at normalizeUnit (duration.js:324)
    at normalizeObject (util.js:259)
    at Function.fromObject (duration.js:245)
    at Function.fromISO (duration.js:273)
    at Function.fromISO (interval.js:158)
    at <anonymous>:1:16
```

**To Reproduce**

```js
luxon.Interval.fromISO('2007-03-01T13:00:00Z/P1Y2M10DT2H30M', {setZone: true}).toString()
```

**Actual vs Expected behavior**

Should have parsed `text` correctly, setting zone to UTC.

**Desktop (please complete the following information):**

- OS: macOS 10.15
- Browser: Chrome 87
- Luxon version: 1.25.0

**Additional context**

Perhaps related to https://github.com/moment/luxon/issues/493

Thank you!

**Comment by icambron** [comment]
hmm, yeah, we should fix that

**Comment by rgroothuijsen** [comment]
Would it be sufficient to add `setZone` as a non-unit key [here](https://github.com/moment/luxon/blob/master/src/duration.js#L245)?

**Comment by socraticDevBlog** [comment]
Cannot reproduce the bug on luxon version 2.4.0

I tried to reproduce the bug against current codebase(by unit test) then in Safari on luxon's website `https://moment.github.io/luxon`.

Do you maintain (fix bugs) in version 1.25.0 reported by @dmalan  ?

<img width="609" alt="image" src="https://user-images.githubusercontent.com/41236715/168184309-d44bdac8-f466-4d8c-82e3-dd3432b86bd0.png">
